### PR TITLE
feat: Add Spanish localization

### PR DESCRIPTION
**Pull request by paolovalerdi**

### CODE REVIEW

File: lib/src/localization.dart [hunk 1]
```diff
@@ -46,6 +47,8 @@ class TimetableLocalizationsDelegate
         return const TimetableLocalizationEn();
       case 'ja':
         return const TimetableLocalizationJa();
+      case 'es':
+        return const TimetableLocalizationEn();
```
Comment (JonasWanke) [inline lib/src/localization.dart:51]:
```suggestion
        return const TimetableLocalizationEs();
```

**Review by JonasWanke — CHANGES_REQUESTED** [review]
Thanks for the contribution! It looks like there's one small typo, but after that is fixed, I'm happy to merge this PR

**Comment by paolovalerdi** [comment]
Oops 😅, I didn't realize, it's fixed, now and I also add it to the supported locales in the example app

**Comment by JonasWanke** [comment]
Thanks for the quick fix :) I'll try to release it soon!

**Comment by JonasWanke** [comment]
Sorry for the delay! This is now published as part of [v1.0.0-alpha.3](https://pub.dev/packages/timetable/versions/1.0.0-alpha.3/changelog#100-alpha3--2021-08-02). Thanks for contributing!

**Pull request merged**